lib/_http_agent.js

```diff
@@ -5,6 +5,8 @@ const util = require('util');
 const EventEmitter = require('events');
 const debug = util.debuglog('http');
 
+const emitError = require('internal/util').emitError;
```

Comment:
This is probably better, but we might as well use:

```
const {emitError} = require('internal/util');
```

Just wondering where we stand on those.